### PR TITLE
Fix #6451 ContextInfoBar Translation not loaded

### LINKED ISSUE #6451: The new right sidebar is not included in translations (Open an issue, Get a demo etc)
### Describe the issue

Example for Japanese:
image

### Environment

- Kestra Version: develop

### PR DESCRIPTION
**Pull request by ghost**
### What changes are being made and why?

This PR fixes a bug related to the incorrect usage of the useI18n Composition API in the ContextInfoBar.vue component. Previously, the local instance of useI18n was used without specifying the global scope, resulting in a mismatch between the global language settings and the sidebar translations.

The following changes have been made:

Updated the useI18n import to use useScope: "global", ensuring that the global language settings are applied consistently throughout the application.
Verified that changing the language and reloading the application correctly reflects the updates in the sidebar.
This change addresses an issue where sidebar content did not update to reflect language changes until a reload occurred, ensuring consistency with the global language scope.

---

### How the changes have been QAed?

The changes have been QAed by performing the following steps:

Changed the language in the application settings.
Saved the settings to update the global language scope.
Reloaded the page and confirmed that the right sidebar reflects the updated language without additional user intervention.
No unit tests were modified, as this fix is scoped to a specific component behavior and integrates with the existing i18n framework

---

### Setup Instructions

Closes https://github.com/kestra-io/kestra/issues/6451/.

### CODE REVIEW

**Comment by anna-geller** [comment]
closes https://github.com/kestra-io/kestra/issues/6451

File: ui/src/components/ContextInfoBar.vue [hunk 1]
```diff
@@ -64,7 +64,7 @@
     import {useStore} from "vuex";
     import {useI18n} from "vue-i18n";
 
-    const {t} = useI18n();
+    const {t} = useI18n({ useScope: "global" });
```
Comment (Piyush-r-bhaskar) [inline ui/src/components/ContextInfoBar.vue]:
```suggestion
    const {t} = useI18n({useScope: "global"});
```

Comment (Piyush-r-bhaskar) [reply ui/src/components/ContextInfoBar.vue]: Translation is working but 
- Not being translated as soon as language is selected.
- Translates only after clicking `Save Changes`.
- language earlier not changed to new without a refresh. - it's happening for me could vary for you.

Comment (ghost) [reply ui/src/components/ContextInfoBar.vue]: Looks like you're referring to the below open issue.

https://github.com/kestra-io/kestra/issues/6460

**Review by MilosPaunovic — APPROVED** [review]
Great catch @arpitgupta-it, thanks for this! 🚀 I've just pushed the changes to all the other places it was necessary. 

Just a small note for the future, if you could use **GitHub Keywords** when opening PRs to connect it to the proper issue (to automate the entire process a bit more), it would be super helpful to us. You can read a bit more about it [here](https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/using-keywords-in-issues-and-pull-requests#linking-a-pull-request-to-an-issue) (Anna added closing keyword for this particular issue).

**Pull request merged**